docs/manual/_renderer.py

```diff
@@ -0,0 +1,19 @@
+from typing import Optional, Union
+
+from griffe import dataclasses as dc
+from plum import dispatch
+from quartodoc import MdRenderer
+
+
+class Renderer(MdRenderer):
+    style = 'mystyle'
+
+    @dispatch
+    def signature(
+        self,
+        el: Union[dc.Class, dc.Function],
+        source: Optional[dc.Alias] = None
+    ):
+        # XXX: doesn't work to set this in __init__
+        self.display_name = 'short'
```

Comment:
Can you point where in the docs you found that `short` is a valid value?

Reply:
I can't find it in the docs. I had to dig through their code/examples. I added a comment explaining what it does.